mp/src/public/vgui_controls/CvarSlider.h

```diff
@@ -29,6 +29,8 @@ namespace vgui
 
         bool ShouldAutoApplyChanges() const { return m_bAutoApplyChanges; }
         void SetAutoApplyChanges(bool val) { m_bAutoApplyChanges = val; }
+        bool OnlyIntegers() const { return m_bOnlyIntegers; }
```

Comment:
Nitpick but usually you want a "getter" word as the first, like `IsOnlyIntegers`.